api/src/test/java/org/openmrs/module/insuranceclaims/api/service/BillServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package org.openmrs.module.insuranceclaims.api.service;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.Concept;
+import org.openmrs.Patient;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.insuranceclaims.api.model.Bill;
+import org.openmrs.module.insuranceclaims.api.model.ProcessStatus;
+import org.openmrs.module.insuranceclaims.api.model.ProvidedItem;
+import org.openmrs.module.insuranceclaims.api.mother.BillMother;
+import org.openmrs.module.insuranceclaims.api.mother.ProvidedItemMother;
+import org.openmrs.module.insuranceclaims.api.util.TestConstants;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.Matchers.is;
+
+public class BillServiceTest extends BaseModuleContextSensitiveTest {
+
+    @Autowired
+    private ProvidedItemService providedItemService;
+
+    @Autowired
+    private BillService billService;
+
+    @Test
+    public void getProvidedItems_shouldCorrectlyGetItemsForPatient() {
+
+        Patient patient = Context.getPatientService().getAllPatients().get(0);
```

Comment:
do not use context

Reply:
.get(0) why ?

Reply:
Why we shouldn't use the Context? Sometimes it is even faster and safer than using the Autowired or XML definition. Btw this is the test and we shouldn't spend to much time on making this beautiful. 

About .get(0) I assume that this is some default patient created by the default test migrations. It will be better to use the getPatient(Integer) method and create some constants for id = 1.